latest-updates/Today-10-09-2023.md

```diff
@@ -0,0 +1,7 @@
+### DROMARA HUTOOL 5.8.21 JSONOBJECT.PUTBYPATH BUFFER OVERFLOW
+
+A vulnerability classified as critical has been found in Dromara HuTool 5.8.21. Affected is the function jsonObject.putByPath. The manipulation with an unknown input leads to a buffer overflow vulnerability. CWE is 
+classifying the issue as CWE-120. The program copies an input buffer to an output buffer without verifying that the size of the input buffer is less than the size of the output buffer, leading to a buffer overflow. 
+This is going to have an impact on confidentiality, integrity, and availability.
+
```

Comment:
Could you please add NEW changes in this pull request file name Today-10-09-2023.md. It will be very helpful.

Reply:
ok